RELEASE_NOTES.md

```diff
@@ -1,11 +1,13 @@
-The RHPAM Kogito Operator is based on Kogito Operator. 
-Please, see the upstream [RELEASE NOTES](https://github.com/kiegroup/kogito-operator/blob/master/RELEASE_NOTES.md).
-
-<!-- Remove the comment for any RHPAM JIRA exclusive 
-## Enhancements  
+The RHPAM Kogito Operator is based on Kogito Operator. Please see the upstream [RELEASE NOTES](https://github.com/kiegroup/kogito-operator/blob/master/RELEASE_NOTES.md).
 
 ## Bug Fixes
 - [KOGITO-4894](https://issues.redhat.com/browse/KOGITO-4894) - Remove `image` default value in KogitoRuntime CR (problem in OCP console)
 - [KOGITO-5104](https://issues.redhat.com/browse/KOGITO-5104) - Kogito operator creates wrong default image tags
```

Comment:
Those JIRAs don't seem to be related any more with next release, could you please remove them?